### PR TITLE
ACN_CD_SymantecEndpointProtection_Workbook_01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Symantec Endpoint Protection Workbook
  - 2x screenshots
  - WorkbookMetadata updates

### CODE REVIEW

**Review by v-maudan — DISMISSED** [review]

**Review by v-jayakal — APPROVED** [review]

**Comment by v-jayakal** [comment]
@Amitbergman, @liatlishams  - Can you please review the PR? Thanks.

**Comment by v-jayakal** [comment]
Can you please change the value of fromTemplateId to "sentinel-SymantecEndpointProtection" as it should be sentinel-workbookname. Please refer the documentation here https://github.com/Azure/Azure-Sentinel/blob/45310fe6be3cb30cbd8f073443a0c2f9144115fa/Workbooks/README.md

**Comment by chicduong** [comment]
updates made

**Review by v-jayakal — DISMISSED** [review]

**Comment by v-jayakal** [comment]
@chicduong - Can you resolve the conflicts? Thanks.

File: Workbooks/SymantecEndpointProtection.json [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+        "value": "RiskEvents"
+      },
+      "customWidth": "40",
+      "name": "Top 10 User Names"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/7ac4b7c7-65f2-46c7-87bb-ca6ac8e5dbef/resourcegroups/socaas-rg/providers/microsoft.operationalinsights/workspaces/toncoso"
+  ],
+  "fromTemplateId": "sentinel-SymantecEndpointProtection",
```
Comment (chicduong) [inline Workbooks/SymantecEndpointProtection.json:807]: changes made

Comment (chicduong) [reply Workbooks/SymantecEndpointProtection.json:807]: @v-jayakal are these the correct changes?

Comment (v-jayakal) [reply Workbooks/SymantecEndpointProtection.json:807]: Yes.

**Comment by v-jayakal** [comment]
@chicduong - There are conflicts in WorkbookMetadata.json, Can you please resolve them for approval and merging? Thanks.

File: Workbooks/SymantecEndpointProtection.json (hunk 1 above)
Comment (chicduong) [reply Workbooks/SymantecEndpointProtection.json:807]: > 
> 
> @chicduong - There are conflicts in WorkbookMetadata.json, Can you please resolve them for approval and merging? Thanks.

@v-jayakal forgive me, I'm not seeing the conflicts in the WorkbookMetatdata.json. They seem to match the require content in the example

**Comment by chicduong** [comment]
> 
> 
> @chicduong - There are conflicts in WorkbookMetadata.json, Can you please resolve them for approval and merging? Thanks.

I believe I have made the correct updates. Let me know if theyre incorrect.

**Review by v-maudan — APPROVED** [review]

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**